src/multilspy/multilspy_utils.py

```diff
@@ -6,6 +6,7 @@
 import logging
 import os
 from typing import Tuple
+from typing_extensions import Union
```

Comment:
Is this not present in typing?

Reply:
It is, fixed 👍 

I believe I've noticed it elsewhere in the codebase and got under impression that package supports some very old version of Python:
https://github.com/microsoft/multilspy/blob/db809145c4f516e07ef88801b5137e48430a47c5/src/multilspy/multilspy_types.py#L8

Reply:
Yes, eventually this needs to be cleaned.